recipes-ni/nilrt-snac/nilrt-snac_git.bb

```diff
@@ -10,23 +10,38 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=380df876633ca23587b9851600778cc0"
 
 SRC_URI = "\
 	git://github.com/ni/nilrt-snac;branch=master;protocol=https \
+	file://run-ptest \
 "
 
-
 SRCREV = "${AUTOREV}"
 PV = "0.1.1+git${SRCPV}"
 
 S = "${WORKDIR}/git"
 
 
+inherit ptest
+
+
 do_install() {
 	oe_runmake install \
 		DESTDIR=${D}
 }
 
+do_install_ptest() {
+	install -m 0755 ${WORKDIR}/run-ptest ${D}${PTEST_PATH}
+}
+
 
 RDEPENDS:${PN} = "\
 	bash \
 	opkg \
 	python3-core \
 "
+
+FILES:${PN}-ptest += "${libdir}/${PN}/tests/integration"
+RDEPENDS:${PN}-ptest = "\
```

Comment:
[ptest.bbclass ](https://github.com/ni/openembedded-core/blob/nilrt/master/next/meta/classes-recipe/ptest.bbclass#L26C1-L26C21)already adds RDEPENDS on ${PN}. Probably better to use a `+=` here and remove `nilrt-snac`.

Reply:
Will fix.

Reply:
Any reason to introduce a new commit to fix this rather than update the original commit 0b61b17445a45cd704f2dc1820423159f544b282?

Reply:
Because there was nothing broken about the old commit - it was just suboptimal, making a new commit is slightly more clear in the PR than force pushing new commit hashes.